packages/client/src/components/communicator/webBased/encoding.ts

```diff
@@ -30,8 +30,8 @@ export function decodeResponseURLParams(params: URLSearchParams): RPCResponseMes
     const { iv, cipherText } = contentParam.encrypted;
     content = {
       encrypted: {
-        iv: hexToBytes(iv),
-        cipherText: hexToBytes(cipherText),
+        iv: new Uint8Array(Buffer.from(iv, 'base64')),
```

Comment:
Does noble hashes not have a util for this?

Reply:
Doesn't look like it